site/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: site
+resources:
+  - namespace.yaml
+  - resources.yaml
+  - networkpolicy.yaml
+replicas:
+  - name: site
+    count: 2
+images:
+  - name: hashbang/site
```

Comment:
cc @daurnimator why are we using this one instead of the hashbang/hashbang.sh Docker autobuild? Also, is the hashbang.sh site ready for an HTTPS-only mode?